webpack.config.js

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Microsoft ?

Reply:
Yeah the webpack configuration as shown in the official documentation for vscode extension has the Microsoft license and even vscode-java uses the same. So I preferred the same.